sdk/proto/analyzer.proto

```diff
@@ -116,13 +116,15 @@ message AnalyzeDiagnostic {
     repeated string tags = 6;              // Keywords/terms to associate with a policy, e.g., "cost".
     EnforcementLevel enforcementLevel = 7; // Severity of the policy violation.
     string urn = 8;                        // URN of the resource that violates the policy.
+    string policyPackVersionTag = 9;       // Version Tag of the Policy Pack.
 }
 
 // AnalyzerInfo provides metadata about a PolicyPack inside an analyzer.
 message AnalyzerInfo {
 	string name = 1;                  // Name of the PolicyPack.
 	string displayName = 2;           // Pretty name for the PolicyPack.
 	repeated PolicyInfo policies = 3; // Metadata about policies contained in PolicyPack.
+    string versionTag = 4;            // Version of the Policy Pack.
```

Comment:
Nit: The alignment looks like its off by two spaces here.

Reply:
I noticed this but on my VS code it looks aligned 🤔